Add nine.ch and datadog sponsoring to website (#543)
voxpupuli/voxpupuli.github.io: _sponsors/nine.md

---
name: Nine
logo: logo-nine.png
website: https://nine.ch/
button: nine.ch
---

> Nine is a cloud provider from Switzerland. They offer a wide range from virtual machines, over dedicated servers to hosted kubernetes. They support us with 250$/month!

Reviewer: ```suggestion
Nine is a cloud provider from Switzerland.
They offer a wide range of products, from virtual machines, to dedicated servers, to hosted Kubernetes.
They support Vox Pupuli with $250/month!
```

Reviewer: Did in #544.
